bHY6yh-de3258/cod-bxxqqq_bHY6yh-de3258.xml

```diff
@@ -0,0 +1,236 @@
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#B" n="6-v"/>
+            <cb ed="#B" n="a"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-de3258"><!-- p1l1-r12 -->
+            <head xml:id="bHY6yh-de3258-Hd1e108">Rubrica 12</head>
+            <head xml:id="bHY6yh-de3258-Hd1e111" type="question-title">De timore Dei</head>
+            <p xml:id="bHY6yh-de3258-d1e114">
+                De timore dei. Rubrica. 12
+    </p>
+            <p xml:id="bHY6yh-de3258-d1e119">
+                <ref xml:id="bHY6yh-de3258-Rd1e116">IOb hystorio capitulo 28</ref> <quote xml:id="bHY6yh-de3258-Qd1e123">Timor domini
+                <lb ed="#B"/>ipse est sapientia.</quote> 
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e129">
+               <lb ed="#B"/>
+                <ref xml:id="bHY6yh-de3258-Rd1e128">Moyses in deuteronomio capitulo 6</ref> <quote xml:id="bHY6yh-de3258-Qd1e133"> <!--line jumps up--> Timeas <!--line jumps down--> dominum 
+                <lb ed="#B"/>deum tuum et custodias omniam man<lb ed="#B" break="no"/>data 
+                et precepta eius et cetera et cetera.</quote>
+            <pb ed="#B" n="6-v"/>
+            <cb ed="#B" n="b"/>
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e142">
+              <lb ed="#B"/>
+                <ref xml:id="bHY6yh-de3258-Rd1e144">Psalmista psalmus 2</ref> <quote xml:id="bHY6yh-de3258-Qd1e146">Seruite domino 
+                <lb ed="#B"/>in timore et exultate ei cum tremore. </quote>
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e152">
+      <lb ed="#B"/> ¶ <ref xml:id="bHY6yh-de3258-Rd1e154">psalmus 28</ref> <quote xml:id="bHY6yh-de3258-Qd1e154"> firmamentum est dominus timen<lb ed="#B" break="no"/>tibus eum</quote>
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e160">
+        ¶ <ref xml:id="bHY6yh-de3258-Rd1e164">32 psalmus</ref>
+        <quote xml:id="bHY6yh-de3258-Qd1e162">Oculi domini super ti<lb ed="#B" break="no"/>mentes 
+        eum </quote>
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e168">
+       ¶<ref xml:id="bHY6yh-de3258-Rd1e173">psalmus 88</ref> <quote xml:id="bHY6yh-de3258-Qd1e170">
+              Prope timentes 
+              <lb ed="#B"/>deum salutare ipsius</quote>
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e176">
+            omitted
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e186">
+      ¶ <quote xml:id="bHY6yh-de3258-Qd1e188">psalmus 110 Ini
+      <lb ed="#B"/>tium sapientie timor domini. </quote>
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e195">
+       ¶<ref xml:id="bHY6yh-de3258-Rd1e204">psalmus 111</ref> <quote xml:id="bHY6yh-de3258-Qd1e197">Be<lb ed="#B" break="no"/>atus vir qui timet dominum. </quote>
```

Comment:
check your spacing and indenting here. We should have something like:
```
<p>
   ¶ <ref>...
</p>
 
 ```

Also at the end of this line after: 
"Be<lb ed="#B" break="no"/>atus" we should get a hard return so that we have the following:

```
<p>
  ¶ <ref xml:id="bHY6yh-de3258-Rd1e204">psalmus 111</ref> <quote>Be<lb ed="#B" break="no"/>atus 
  vir qui timet dominum. </quote>
</p>
```

Generally check the indenting throughout. For each paragraph block we want every line to be indented by two spaces, so that we have the following: 

```
<p>
  <lb/>text text text
  <lb/>text text text
  <lb/>text text te<lb break="no"/>xt
  <ref>text text</ref> text text
  <lb/>text text text
  <lb/>text text te<lb break="no"/>xt
  text text text text
  <lb/>text text text
</p>